common/src/main/java/tds/common/configuration/SecurityConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package tds.common.configuration;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+import static org.springframework.boot.autoconfigure.security.SecurityProperties.ACCESS_OVERRIDE_ORDER;
+
+/**
+ * This security configuration allows access to the /info.json and /health.json endpoints,
+ * which are required for health-check requests that don't include an Accept header.
+ */
+@Configuration
+@EnableWebSecurity
+@Order(ACCESS_OVERRIDE_ORDER)
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+
+    @Override
+    protected void configure(final HttpSecurity http) throws Exception {
+        http
+            .authorizeRequests()
+            .antMatchers("/info*", "/health*").permitAll();
+    }
+
+}
```

Comment:
This security configurer leaves all endpoints unprotected (as before) except for the Spring Actuator endpoints, which are now protected.  See ManagementWebSecurityAutoConfiguration if you want to dive into the spring magick.